cmd/fyne/commands/deprecated.go

```diff
@@ -0,0 +1,45 @@
+package commands
+
+import (
+	"errors"
+	"fmt"
+	"os"
+	"os/exec"
+	"runtime/debug"
+
+	"github.com/urfave/cli/v2"
+)
+
+// Vendor returns the vendor cli command
```

Comment:
Nit:

I'm not going to blag every occurrence of this, just the first in each package.

Apparently code comments are supposed to end with a period if they are not to start a heading.

```suggestion
// Vendor returns the vendor cli command.
```

From go blog: https://blog.golang.org/godoc

> Notice this comment is a **complete sentence** that begins with the name of the element it describes. This important convention allows us to generate documentation in a variety of formats, from plain text to HTML to UNIX man pages, and makes it read better when tools truncate it for brevity, such as when they extract the first line or sentence.

From godoctricks: https://godoc.org/github.com/fluhus/godoc-tricks#Paragraphs

Searching through official documentation on the standard library we can also see this standard in practice:

https://golang.org/src/archive/tar/common.go

Reply:
Sure. A pretty picky thing given that a bunch of comments within Fyne doesn't do that, but I will get that updated.